browser_patches/prepare_checkout.sh

```diff
@@ -31,18 +31,20 @@ FRIENDLY_CHECKOUT_PATH="";
 CHECKOUT_PATH=""
 PATCHES_PATH=""
 BUILD_NUMBER=""
-PLAYWRIGHT_PATH=""
+WEBKIT_EXTRA_FOLDER_PATH=""
+FIREFOX_EXTRA_FOLDER_PATH=""
 if [[ ("$1" == "firefox") || ("$1" == "firefox/") || ("$1" == "ff") ]]; then
```

Comment:
nit: perhaps it makes sense to share some code between prepare_checkout and export?

Reply:
i'd keep it for now.